apps/back/src/Twig/AppExtension.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class AppExtension extends AbstractExtension
+{
+    public function getFilters(): array
+    {
+        return array(
+            new TwigFilter('divide_by_100', array($this, 'divideBy100')),
+        );
```

Comment:
FYI you can use the following syntax which is more commonly used:

```suggestion
        return [
            new TwigFilter('divide_by_100', array($this, 'divideBy100')),
        ];
```